deploy-board/deploy_board/templates/configs/env_config.tmpl

```diff
@@ -7,13 +7,16 @@
                 <div class="form-group">
                     <label for="maxParallelHosts" class="deployToolTip control-label col-xs-2"
                         data-toggle="tooltip"
-                        title="Maximum number of hosts allowed to deploy at the same time">
-                        Maximum Hosts
+                        title="Maximum number (or percentage) of hosts allowed to deploy at the same time">
+                        Maximum Deploys In Parallel
                     </label>
 
                     <div class="col-xs-4">
-                        <input class="form-control" name="maxParallelHosts" required="true"
-                               type="text" value="{{ env.maxParallel }}"/>
+                        <div class="input-group">
+                            <input class="form-control" name="maxParallelHosts" required="true"
+                               type="text" value="{{ env.finalParallel }}"/>
+                            <span class="input-group-addon">eg. 5 or 20%</span>
```

Comment:
can we ask add a radiobutton, or dropdown, for users to explicitly choose percentage or absolute numbers? I feel user have to type % in the input box not very elegant